woocommerce/woocommerce
# [WIP][Accessibility] Add field for the email footer text color

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #43603 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open the default Woocommerce > Settings > Email
![image](https://github.com/user-attachments/assets/bf4bec2e-8f52-4de7-95f4-255e930a29f9)

2. There should now see an additional field for Footer text color
![image](https://github.com/user-attachments/assets/fb3051bf-bed5-437f-a6c9-1ac6f7e04e0a)

3. Update the color
4. Preview the email and note that the color is as-is with no reduction in opacity



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Add field for the email footer text color

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
